src/main/java/ch/sbb/maven/plugins/markdown2html/markdown/MarkdownProcessor.java

```diff
@@ -40,6 +44,50 @@ public String removeChapter(@NotNull String markdown, @Nullable String chapterTi
         String result = matcher.replaceAll("");
 
         // Remove leading and trailing empty lines around the remaining text
-        return result.replaceAll("(?m)^\\s*\n", "\n").trim();
+        return result.replaceAll("(?m)^\\s*\\R", "\n").trim();
+    }
+
+    public @NotNull String removeLinesContainedSubstrings(@NotNull String markdown, @Nullable List<String> lineSubstrings) {
```

Comment:
Maybe it would be better to rename it to removeLinesContainingSubstrings

Reply:
Renamed